src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.WorkerThread.cs

```diff
@@ -314,28 +296,15 @@ private static bool TakeActiveRequest(PortableThreadPool threadPoolInstance)
                 return false;
             }
 
-            private static bool TryCreateWorkerThread()
+            private static void TryCreateWorkerThread()
```

Comment:
```suggestion
            private static void CreateWorkerThread()
```
This should not be called "Try" now that it is not handling failures